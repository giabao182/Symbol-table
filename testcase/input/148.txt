INSERT x number
BEGIN
INSERT y number
ASSIGN y 10
ASSIGN x y
END
ASSIGN y 100