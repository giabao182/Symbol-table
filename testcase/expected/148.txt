success
success
success
TypeMismatch: ASSIGN x y
Undeclared: ASSIGN y 100